README.md

```diff
@@ -374,9 +374,8 @@ Other Style Guides
     // bad
     const len = items.length;
     const itemsCopy = [];
-    let i;
 
-    for (i = 0; i < len; i += 1) {
+    for (let i = 0; i < len; i += 1) {
```

Comment:
this would make it a less "bad" example, so i'm not sure why it's an improvement

Reply:
Fair enough.